src/webhint-theme/source/js/telemetry/telemetry.js

```diff
@@ -49,10 +49,12 @@
     const track = async (type, data) => {
         telemetryQueue.push(
             {
-                name: data.name,
-                properties: Object.assign(Object.assign({}, options.defaultProperties), data.properties),
-                ver: 2,
-                type: `${type}Data`
+                data: {
```

Comment:
Note: I created a separate PR and merged this change separately because the telemetry API had already been updated back in June (which was causing us to fail reporting 28-day activity for the online scanner).

Leaving this PR open for the separate changes in server/index.js which require a new webhint release before they can be published.